src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -725,6 +734,15 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
       },
       (errors) => {
         this._errors = errors.body.message;
+        showToast(this, {
+          message: errors.body.message,
+          dismissable: false,
+          duration: 0,
```

Comment:
We should give it a duration now, and remove the action?

Reply:
Ah, then I misunderstood your intent. Changed now.